### DISCUSSION TITLE
What is the difference to the original version from cupcakearmy/autorestic?

### DISCUSSION THREAD

**Discussion by ovizii** [pascaliske/docker-autorestic#76] [Q&A] [answered]
Just curios to know, so I can make an informed decision about which oen to use.

**Comment by pascaliske** [comment, marked as answer]
Hey @ovizii,

Thanks for asking this. This project just [builds a binary from the original repository](https://github.com/pascaliske/docker-autorestic/blob/master/Dockerfile#L15-L23) and [wraps it in a Docker image](https://github.com/pascaliske/docker-autorestic/blob/master/Dockerfile#L38-L40).

Hope this answers your question. 😊

**Comment by ovizii** [comment]
Oh, I see, I thought there was something different seeing that cupcake/autorestic also has a docker image: 

https://hub.docker.com/r/cupcakearmy/autorestic

**Reply by pascaliske** [reply]
Yeah. I think the official image was introduced later on and if I'm honest I'm currently not actively using this project any longer... 😅
